api/market.ts

```diff
@@ -156,3 +160,39 @@ export async function fetchCoupons(
     available,
   }
 }
+
+export async function fetchBorrowAprByEpochsBorrowed(
+  asset: Asset,
```

Comment:
What if we use `substitute: Currency` instead, as other fields of `asset` are not used in this function, and we might want to calculate the APR of different substitutes within the same `asset` in the future?